src/toolbar/toolbar-group.jsx

```diff
@@ -197,7 +196,7 @@ const ToolbarGroup = React.createClass({
     }, this);
 
     return (
-      <div {...other} className={className} style={this.prepareStyles(styles.root, style)}>
+      <div {...other} className={className} style={prepareStyles(Object.assign({}, styles.root, style))}>
```

Comment:
I think more than 50% would be safe, you're right. Let's go with this approach for now :+1: